lighthouse-service/event_handler/common.go

```diff
@@ -8,7 +8,8 @@ import (
 
 	"github.com/ghodss/yaml"
 	utils "github.com/keptn/go-utils/pkg/api/utils"
-	keptn "github.com/keptn/go-utils/pkg/lib"
+	keptnv1 "github.com/keptn/go-utils/pkg/lib"
+	keptn "github.com/keptn/go-utils/pkg/lib/keptn"
```

Comment:
something seems weird with those imports, what's going on here?

Reply:
the structure of the go-utils has changed